lib/rubocop/cop/rspec/match_array.rb

```diff
@@ -20,11 +20,34 @@ module RSpec
       class MatchArray < Base
```

Comment:
ask) How about adding bad/good cases?

Reply:
To documentation? Yeah, I should do that.